test/e2e/customconfigs/skipjobswithoutqueuename_test.go

```diff
@@ -134,5 +129,57 @@ var _ = ginkgo.Describe("ManageJobsWithoutQueueName", ginkgo.Ordered, func() {
 				}, util.Timeout, util.Interval).Should(gomega.Succeed())
 			})
 		})
+
+		ginkgo.It("should suspend a pod created in the test namespace", func() {
+			var testPod, createdPod *corev1.Pod
+			var podLookupKey types.NamespacedName
+			ginkgo.By("creating a pod without a queue name", func() {
+				testPod = testingpod.MakePod("test-pod", ns.Name).Obj()
+				gomega.Expect(k8sClient.Create(ctx, testPod)).Should(gomega.Succeed())
+			})
+
+			ginkgo.By("verifying that the pod is gated", func() {
+				createdPod = &corev1.Pod{}
+				podLookupKey = types.NamespacedName{Name: testPod.Name, Namespace: ns.Name}
+				gomega.Eventually(func(g gomega.Gomega) {
+					g.Expect(k8sClient.Get(ctx, podLookupKey, createdPod)).Should(gomega.Succeed())
+					g.Expect(createdPod.Spec.SchedulingGates).Should(gomega.BeComparableTo([]corev1.PodSchedulingGate{
+						{Name: "kueue.x-k8s.io/admission"},
+					}))
+					g.Expect(createdPod.ObjectMeta.Labels).To(gomega.BeComparableTo(map[string]string{constants.ManagedByKueueLabel: "true"}))
+					g.Expect(createdPod.Finalizers).Should(gomega.ContainElement(constants.ManagedByKueueLabel))
+				}, util.Timeout, util.Interval).Should(gomega.Succeed())
+			})
+		})
+
+		ginkgo.It("should NOT suspend a pod created in the kube-system namespace", func() {
+			var testPod, createdPod *corev1.Pod
+			var podLookupKey types.NamespacedName
+			ginkgo.By("creating a pod without a queue name", func() {
+				testPod = testingpod.MakePod("test-pod", "kube-system").
+					Image(util.E2eTestSleepImage, []string{"1s"}).
+					Request("cpu", "1").
+					Request("memory", "2G").
+					Obj()
+				gomega.Expect(k8sClient.Create(ctx, testPod)).Should(gomega.Succeed())
+			})
+
+			ginkgo.By("verifying that the pod is running", func() {
+				createdPod = &corev1.Pod{}
+				podLookupKey = types.NamespacedName{Name: testPod.Name, Namespace: "kube-system"}
+				gomega.Eventually(func(g gomega.Gomega) {
+					g.Expect(k8sClient.Get(ctx, podLookupKey, createdPod)).Should(gomega.Succeed())
+					g.Expect(createdPod.Status.Phase).Should(gomega.Equal(corev1.PodRunning))
+				}, util.Timeout, util.Interval).Should(gomega.Succeed())
+			})
+
+			ginkgo.By("deleting the pod", func() {
+				gomega.Expect(k8sClient.Delete(ctx, testPod)).Should(gomega.Succeed())
+				gomega.Eventually(func(g gomega.Gomega) {
+					err := k8sClient.Get(ctx, podLookupKey, createdPod)
+					g.Expect(apierrors.IsNotFound(err)).Should(gomega.BeTrue())
+				}, util.Timeout, util.Interval).Should(gomega.Succeed())
```

Comment:
```suggestion
				}, util.LondTimeout, util.Interval).Should(gomega.Succeed())
```
I think we missed this.